src/main/java/uk/gov/justice/hmpps/prison/api/model/AgyPrisonerPayProfileDto.kt

```diff
@@ -0,0 +1,34 @@
+package uk.gov.justice.hmpps.prison.api.model
+import java.math.BigDecimal
+import java.math.RoundingMode
+import java.time.LocalDate
+
+data class AgyPrisonerPayProfileDto(
```

Comment:
not a great fan of abbreviating `Agency` to `Agency`.  Can we just call it `Agency` please?

Reply:
The name of the table in NOMIS is AGY_PRISONER_PAY_PROFILE in this case - abbreviated at source.  Happy either way though.

Reply:
Refactored the model to use the full name, whilst the entity mirrors the table name in NOMIS.